rules/windows/powershell/powershell_script/posh_ps_nishang_malicious_commandlets.yml

```diff
@@ -43,7 +43,7 @@ detection:
             - 'FakeDC'
             - 'FireBuster'
             - 'FireListener'
-            - 'Get-Information'
+            - 'Get-Information ' # otherwise FP with Get-InformationBarrierReportDetails or Get-InformationBarrierReportSummary
```

Comment:
```suggestion
            - 'Get-Information ' # Space at the end is required. Otherwise, we get FP with Get-InformationBarrierReportDetails or Get-InformationBarrierReportSummary
```